docs/coding-guidelines/updating-ref-source.md

```diff
@@ -11,6 +11,6 @@ This document provides the steps you need to take to update the reference assemb
 ## For System.Runtime
```

Comment:
Fix [MD022](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines).

```suggestion
## For System.Runtime

```